cache-config/t3c-apply/torequest/torequest.go

```diff
@@ -893,6 +897,10 @@ func (r *TrafficOpsReq) ProcessConfigFiles() (UpdateStatus, error) {
 		}
 	}
 
+	if 0 < len(r.changedFiles) {
```

Comment:
Nitpick: this is an artifact of C, it isn't necessary in Go, `len(x) > 0` is easier to read and understand.
(Assignment has no return value in Go, so `if x = 0 {` doesn't compile.)